src/main/java/com/deavensoft/timetracker/service/WorkLogServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.deavensoft.timetracker.service;
+
+import com.deavensoft.timetracker.api.mapper.WorkLogMapper;
+import com.deavensoft.timetracker.api.model.WorkLogDto;
+import com.deavensoft.timetracker.domain.WorkLog;
+import com.deavensoft.timetracker.repository.WorkLogRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+
+public class WorkLogServiceImpl implements WorkLogService {
+
+    private final WorkLogRepository workLogRepository;
+    private final WorkLogMapper mapper;
+
+    public WorkLogServiceImpl(WorkLogRepository workLogRepository, WorkLogMapper mapper) {
+        this.workLogRepository = workLogRepository;
+        this.mapper = mapper;
+    }
+
+    @Override
+    public WorkLogDto getWorkLogById(Long id) {
+        return workLogRepository
+                .findById(id)
+                .map(mapper::workLogToWorkLogDto)
+                .orElseThrow(RuntimeException::new);
```

Comment:
We should avoid throwing `RuntimeException`s, since it's too generic and be more specific. E.g. in this case you can throw `IllegalArgumentExcpetion` or to create a separate custom exception like `EntityNotFound` or similar.